src/Administration/Resources/app/administration/build/plugins.vite.ts

```diff
@@ -34,6 +34,7 @@ const isDev = VITE_MODE === 'development';
 
 // This env variable is provided by the symfony recipes
 const hasAdminRootEnv = !!process.env.ADMIN_ROOT;
+const host = process.env.HOST || 'localhost';
```

Comment:
can we name this `VITE_HOST` or something like that.

Reply:
This would be possible